govcd/nsxt_edgegateway_test.go

```diff
@@ -100,3 +100,88 @@ func (vcd *TestVCD) Test_NsxtEdgeCreate(check *C) {
 	err = updatedEdge.Delete()
 	check.Assert(err, IsNil)
 }
+
+func (vcd *TestVCD) Test_NsxtEdgeVdcGroup(check *C) {
+	skipNoNsxtConfiguration(vcd, check)
+	skipOpenApiEndpointTest(vcd, check, types.OpenApiPathVersion1_0_0+types.OpenApiEndpointEdgeGateways)
+
+	adminOrg, err := vcd.client.GetAdminOrgByName(vcd.config.VCD.Org)
```

Comment:
usually, we check if err is nill and the returned value is not nill. In all places - just for the sake to void panics in later checks.

Reply:
Added extra checks in quite a few places and reran affected tests.